src/EnrollWebauthnController.js

```diff
@@ -129,30 +100,33 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, FidoUtil, webauthn
       formChildren: function () {
         var children = [];
 
-        if (webauthn.isWebauthnOrU2fAvailable()) {
-          //There is html in enroll.u2f.general2 in our properties file, reason why is unescaped
+        if (webauthn.isNewApiAvailable()) {
+          //enroll.webauthn.instructions.edge is unescaped because it contains html
           children.push(FormType.View({
-            View:
-              '<div class="u2f-instructions">\
-                 <ol>\
-                   <li>{{{i18n code="enroll.u2f.general2" bundle="login"}}}</li>\
-                   <li>{{i18n code="enroll.u2f.general3" bundle="login"}}</li>\
-                 </ol>\
-               </div>\
-               <div class="u2f-enroll-text hide">\
-                 <p>{{i18n code="enroll.u2f.instructions" bundle="login"}}</p>\
-                 <p>{{i18n code="enroll.u2f.instructionsBluetooth" bundle="login"}}</p>\
-                 <div data-se="u2f-devices" class="u2f-devices-images">\
-                   <div class="u2f-usb"></div>\
-                   <div class="u2f-bluetooth"></div>\
-                 </div>\
-                 <div data-se="u2f-waiting" class="okta-waiting-spinner"></div>\
-               </div>'
+            View: Okta.View.extend({
+              className: 'webauthn-enroll-text',
+              template: '\
+                <div class="webauthn-enroll-instructions">\
+                  <p>{{i18n code="enroll.webauthn.instructions" bundle="login"}}</p>\
+                </div>\
+                {{#if isEdge}}\
+                  <div class="webauthn-edge-text">\
+                     <p>{{{i18n code="enroll.webauthn.instructions.edge" bundle="login"}}}</p>\
+                  </div>\
+                {{/if}}\
+                <div data-se="webauthn-waiting" class="okta-waiting-spinner hide"></div>\
+              ',
+              getTemplateData: function () {
+                return {
+                  isEdge: window.navigator.userAgent.indexOf('Edge') > -1
```

Comment:
let's add this as a function to [BrowserFeatures.js](https://github.com/okta/okta-signin-widget/blob/master/src/util/BrowserFeatures.js), similar to what we have there. (also this could simplify how to mock the tests that need the agent to be Edge)